assets/src/edit-story/components/toggleButton/index.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import {
+  Button as dsButton,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  BUTTON_SIZES,
+  Text,
+  TOOLTIP_PLACEMENT,
+  Tooltip,
+} from '../../../design-system';
+
+const Button = styled(dsButton)`
+  height: 36px;
+  width: 58px;
+  display: inline-block;
+  border: 1px solid ${({ theme }) => theme.colors.border.defaultNormal};
+  padding: 2px 0;
+  color: ${({ theme }) => theme.colors.fg.primary};
+  ${({ isOpen, theme }) =>
+    isOpen &&
+    css`
+      border-color: ${theme.colors.bg.secondary};
+      background-color: ${theme.colors.bg.secondary};
+    `}
+`;
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+`;
+
+const NotificationCount = styled(Text).attrs({ as: 'span' })`
+  margin: -2px 0 -2px;
+  padding-right: 6px;
+  line-height: 20px;
+  color: ${({ theme }) => theme.colors.violet[20]};
```

Comment:
Feels like a one-off color usage, so i just referenced the `violet` value itself. If anyone has ideas of what to name this and want to include it in `colors.fg` i'm open to it, just didn't feel relevant.

Reply:
I think this is fine. I don't think it's necessary to make up a new abstraction for a one-off use case